man/coap_pdu_setup.txt.in

```diff
@@ -98,6 +109,9 @@ options.  Note that updating the response pdu's code variable will cause the
 response pdu to get transmitted.  If code does not get updated, then the
 response pdu is freed off by the underlying library.
 
+For handling situations where the data to be transmitted does not fit into a
+single packet, see coap_block(3),
```

Comment:
This sentence seems to be incomplete.

Reply:
Trailing comma replaced with a full-stop.